.github/workflows/build.yaml

```diff
@@ -44,10 +44,14 @@ jobs:
           fi
   build:
     needs: setup
+    continue-on-error: ${{ matrix.experimental }}
     strategy:
       matrix:
         runner: [windows-large, macos-12-xl, linux-large]
```

Comment:
`linux-large` should probably be removed from the main runner array since it's included below.

Reply:
I kind of prefer to have it in both places. not sure what's best.  this seems to be the only way to override some matrix variables for a single platform unfortunately